src/lib/SignatureHelper.ts

```diff
@@ -0,0 +1,110 @@
+import { soliditySha3 } from 'web3-utils';
+import { stripHexPrefix } from './BytesHelper';
+import { address } from '../../src/types';
+
+export const SIGNATURE_TYPES = {
+  INVALID: 0,
+  NO_PREPEND: 1,
+  DECIMAL: 2,
+  HEXADECIMAL: 3,
+  UNSUPPORTED: 4,
+};
+
+export const PREPEND_DEC = '\x19Ethereum Signed Message:\n32';
```

Comment:
you can also type more of these things e.g. `export const PREPEND_DEC: string = ...`